config/templates/managed-velero-operator-csv-template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: managed-velero-operator-0.0.1
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: "false"
+    description: Operator to manage installation of Velero in OpenShift.
+    containerImage: quay.io/app-sre/managed-velero-operator:latest
+    createdAt: "2019-11-11T11:00:00Z"
+    support: Stefanie Forrester
```

Comment:
Do we put individuals names in here?

Reply:
Yes, I copied this from other operators. Though I committed this is back when you were on a different team, so now that you're back I'd be happy to put your name there instead :)

```
[dakini@nibbana openshift]$ find -name "*-csv-template.yaml" -exec grep "support: " '{}' \;
    support: Sam Nguyen
    support: Devan Goodwin
    support: Matt Woodson
    support: Carl Brumm
    support: Stefanie Forrester
    support: Wesley Hearn
    support: James Harrington
[dakini@nibbana openshift]$ find -name "*-csv-template.yaml"
./rbac-permissions-operator/config/templates/rbac-permissions-operator-csv-template.yaml
./hive/config/templates/hive-csv-template.yaml
./deadmanssnitch-operator/config/templates/deadmanssnitch-operator-csv-template.yaml
./certman-operator/config/templates/certman-operator-csv-template.yaml
./managed-velero-operator/config/templates/managed-velero-operator-csv-template.yaml
./pagerduty-operator/config/templates/pagerduty-operator-csv-template.yaml
./aws-account-operator/config/templates/aws-account-operator-csv-template.yaml
```